Realm/ObjectServerTests/SwiftSyncTestCase.swift

```diff
@@ -324,6 +164,24 @@ open class SwiftSyncTestCase: RLMSyncTestCase {
                 file: (fileName), line: lineNumber)
         }
     }
+
+    public static let bigObjectCount = 2
+    public func populateRealm(user: User, partitionValue: String) {
```

Comment:
This parameter is not used. It should either be optional and set on nil or removed entirely.